llvm-runtimes/libcxx/libcxx-22.0.0.9999.ebuild

```
@@ -191,6 +191,18 @@ multilib_src_install() {
 		dolib.so lib/libc++_shared.so
 		use static-libs && dolib.a lib/libc++_static.a
 	fi
+
+	insinto /usr/share/libc++/gdb
+	doins ../libcxx/utils/gdb/libcxx/printers.py
+
+	local gdb_autoload_dir="/usr/share/gdb/auto-load/usr/$(get_libdir)"
+	dodir "${gdb_autoload_dir}"
+	local lib_version=$(grep '^LIBCXX_LIBRARY_VERSION:STRING=' CMakeCache.txt | cut -d'=' -f2)
+	[[ -n ${lib_version} ]] || die "Could not determine LIBCXX_LIBRARY_VERSION from CMakeCache.txt"
+	cat <<-EOF > "${ED}${gdb_autoload_dir}/libc++.so.${lib_version}-gdb.py" || die
```

```suggestion
	newins - "libc++.so.${lib_version}-gdb.py" <<-EOF
```

And `insinto` appropriately.